wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -193,6 +194,12 @@ executable cardano-node
                        -Wall
                        -O2
   build-depends:       base
+                     , QuickCheck
+                     , aeson
+                     , aeson-pretty
+                     , acid-state
+                     , bytestring
+                     , conduit
```

Comment:
to make locating deps easier, would you be willing to sort this list? in vim you can do it by visual-selecting all the lines under `base` and doing `:! sort`